src/dune/docker.py

```diff
@@ -97,6 +97,18 @@ def cp_from_host(self, host_file, container_file):
     def rm_file(self, file_name):
         self.execute_cmd(['rm', '-rf', file_name])
 
+    def write_file(self, file_name, body):
+        return self.execute_cmd(['echo', body, '>', file_name])
+
+    def append_file(self, file_name, body):
+        return self.execute_cmd(['echo', body, '>>', file_name])
+
+    def get_arch(self) :
+        stdout, stderr, exit_code = self.execute_cmd(['uname', '-m'])
+        if stdout in ('x86_64\n', 'amd64\n') :
+            return 'amd64'
+        return 'arm64'
```

Comment:
anything that's not intel or amd 64 is arm? I don't have a better suggestion so it is what it is.

Reply:
Yeah, I'm not super enthused about it either. But, I couldn't prove to myself that all ARM systems would return aarch64, I read that some return arm8, arm9, etc.  I believe it should also be aarch64 for 64bit ARM, but once again I couldn't find definitive information and I didn't want to go off of the one Mac Mini as a 'proof' that that works.

Plus, we explicitly own the packages for these and only support arm64 or amd64.